packages/docs/docs/guides/3-Portal-API/5-api-conclusion.md

```diff
@@ -9,10 +9,18 @@ description: >-
 ---
 
 
-Need to explain how to connect their app: https://dashboard.request.network/settings/apps
+Request API is an abstraction over the Request Network for easy, but centralised, access. It's the fastest way to get started and build integrations.
+
 ## Pros
-TODO
+* Simple REST API, familiar for classic web developers
+* No key management
+* Faster response time
```

Comment:
With the buffer on the storage layer I'm not sure it's still the case. But I'm ok with leaving this here until we have measurements.

Reply:
read performance is still better.